fabric-tree-api-v1/src/main/java/net/fabricmc/fabric/api/tree/v1/FabricTreeRegistry.java

```diff
@@ -0,0 +1,125 @@
+package net.fabricmc.fabric.api.tree.v1;
+
+import com.mojang.serialization.Codec;
+import net.fabricmc.fabric.mixin.tree.BlockStateProviderTypeInvoker;
+import net.fabricmc.fabric.mixin.tree.FoliagePlacerTypeInvoker;
+import net.fabricmc.fabric.mixin.tree.TreeDecoratorTypeInvoker;
+import net.fabricmc.fabric.mixin.tree.TrunkPlacerTypeInvoker;
+import net.minecraft.util.Identifier;
+import net.minecraft.util.math.intprovider.IntProvider;
+import net.minecraft.util.math.intprovider.IntProviderType;
+import net.minecraft.world.gen.foliage.FoliagePlacer;
+import net.minecraft.world.gen.foliage.FoliagePlacerType;
+import net.minecraft.world.gen.stateprovider.BlockStateProvider;
+import net.minecraft.world.gen.stateprovider.BlockStateProviderType;
+import net.minecraft.world.gen.treedecorator.TreeDecorator;
+import net.minecraft.world.gen.treedecorator.TreeDecoratorType;
+import net.minecraft.world.gen.trunk.TrunkPlacer;
+import net.minecraft.world.gen.trunk.TrunkPlacerType;
+
+/**
+ * <p>
+ * 	   A registry for {@link TrunkPlacerType}s, {@link FoliagePlacerType}s, {@link TreeDecoratorType}s,
+ * 	   {@link BlockStateProviderType}s and {@link IntProviderType}s.
```

Comment:
Use an Oxford comma here:

```suggestion
 * 	   {@link BlockStateProviderType}s, and {@link IntProviderType}s.
```

Reply:
I'm a Russian learning English (B2), so don't judge me :)